src/lib/hooks/useTokenList/index.ts

```diff
@@ -44,6 +46,19 @@ export default function useTokenList(list?: string | TokenInfo[]): WrappedTokenI
   }, [chainId, chainTokenMap])
 }
 
+export type TokenMap = { [address: string]: Token }
+
+export function useTokenMap(): TokenMap {
```

Comment:
Non-blocking - but it kinda confuses me that this is also called TokenMap but its different that the type TokenMap in utils below